src/main/java/net/patchworkmc/patcher/patch/KeyBindingsTransformer.java

```diff
@@ -0,0 +1,46 @@
+package net.patchworkmc.patcher.patch;
+
+import org.objectweb.asm.ClassVisitor;
+import org.objectweb.asm.MethodVisitor;
+import org.objectweb.asm.Opcodes;
+
+public class KeyBindingsTransformer extends ClassVisitor {
+	// The intermediary name for KeyBinding
+	private static final String KEY_BINDING = "net/minecraft/class_304";
+
+	// Patchwork's replacement key binding class
+	private static final String PATCHWORK_KEY_BINDING = "net/patchworkmc/impl/keybindings/PatchworkKeybinding";
```

Comment:
This should be in `.api`. Also, would be nice to have `KeyBinding` instead of `Keybinding`.